internal/pkg/addon/addons.go

```diff
@@ -21,7 +21,7 @@ const (
 
 var (
 	wkldAddonsParameterReservedKeys = []string{"App", "Env", "Name"}
-	envAddonsParameterReservedKeys  = []string{"App", "Name"}
+	envAddonsParameterReservedKeys  = []string{"App", "Env"}
```

Comment:
Why do we make this change?

Reply:
I replied to Efe here: https://github.com/aws/copilot-cli/pull/4258#discussion_r1046557439